nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -2920,20 +2920,21 @@
   implicit-imports:
     - post-import-code:
         - |
-          def myconnect(self, func):
+          def patched_connect(self, func, conn_type=None):
+              conn_type = conn_type or QtCore.Qt.ConnectionType.AutoConnection
               if hasattr(func, "im_func"):
                   if hasattr(func.im_func, "__compiled__"):
-                      myconnect._protected = getattr(myconnect, "_protected", set())
-                      myconnect._protected.add(func)
+                      patched_connect._protected = getattr(patched_connect, "_protected", set())
```

Comment:
I agree, `my` prefix is a choice that works for private projects, but must have been confusing.